cms/methodology/models.py

```diff
@@ -0,0 +1,91 @@
+from typing import TYPE_CHECKING, Any, ClassVar
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.functional import cached_property
+from django.utils.translation import gettext_lazy as _
+from wagtail.admin.panels import FieldPanel, InlinePanel, MultiFieldPanel
+from wagtail.fields import RichTextField
+from wagtail.search import index
+
+from cms.core.blocks.stream_blocks import SectionStoryBlock
+from cms.core.fields import StreamField
+from cms.core.models import BasePage
+
+if TYPE_CHECKING:
+    from django.http import HttpRequest
+    from wagtail.admin.panels import Panel
+
+
+class MethodologyPage(BasePage):  # type: ignore[django-manager-missing]
+    parent_page_types: ClassVar[list[str]] = ["topics.TopicPage"]
+
+    template = "templates/pages/methodology_page.html"
+
+    summary = RichTextField(features=settings.RICH_TEXT_BASIC)
+    publication_date = models.DateField()
+    last_revised_date = models.DateField(blank=True, null=True)
+
+    contact_details = models.ForeignKey(
+        "core.ContactDetails",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    content = StreamField(SectionStoryBlock())
+
+    show_cite_this_page = models.BooleanField(default=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("summary"),
+        MultiFieldPanel(
+            [
+                FieldPanel("publication", icon="calendar-date"),
+                FieldPanel(
+                    "last_revised_date",
+                ),
```

Comment:
Also

```suggestion
                FieldRowPanel([
                    FieldPanel("publication_date"),
                    FieldPanel("last_revised_date"),
                ]),
```

with `FieldRowPanel` imported make this a tad nicer

<img width="752" alt="Screenshot 2024-12-11 at 16 22 10" src="https://github.com/user-attachments/assets/db097c2e-9f64-40bf-b0f9-50081a1d0fdf" />

Reply:
✨[f21fbdd](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/f21fbdd8f67f800ee357b4d40491504483d05606)✨

Reply:
Is it worth doing the same for the Analysis page for consistency? It looks slightly off:

<img width="799" alt="image" src="https://github.com/user-attachments/assets/ee9e8bb1-a35c-464c-b484-dae6d2da41c9" />